src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -127,6 +128,10 @@ def _stream_logs(byte_size,
                 # Only scan what's newly read. If nothing is read,
                 # default to 0.
                 min_scan_range = max(new_byte_size - amount_read - 1, 0)
+
+                if firstTime:
+                    print("Build agent acquired, starting to stream logs...\n")
```

Comment:
is this log necessary - I'm not sure it's valuable since you can infer it's being processed if logs are coming out...
also the log is in the wrong spot in the code, it needs to be in the below inner loop, this will output even if no logs are streaming

Reply:
Agreed, not printing this anymore.